README.md

```diff
@@ -1,21 +1,34 @@
-# Bodhi application logic in GraphQL + neDB
+# Prerequisites
+You will need the Qtum client for the OS you are testing on (or building against).
 
-## Get Started
+Download the [Qtum client](https://github.com/qtumproject/qtum/releases) for the correct OS and put the `bin/` folder in the corresponding dir:
+
+    qtum/mac/bin
+    qtum/win64/bin
+    qtum/win32/bin
+    qtum/linux64/bin
+    qtum/linux32/bin
+
+# Run Dev Environment
 1. `git clone https://github.com/bodhiproject/bodhi-graphql.git`
 
 2. `cd bodhi-graphql`
 
 3. `npm install`
 
-4. download `qtum-x.xx.xx-osx64.tar.gz` from https://github.com/qtumproject/qtum/releases and unzip it. copy all the folders inside it to ./qtum
+4. `npm start`
+
+5. App at `127.0.0.1:5555` or GraphiQL at `127.0.0.1:5555/graphiql`
 
-5. start bodhi-graphql at 5555
+# Release
+We are currently using [Electron](https://github.com/electron/electron) and [Electron-Builder](https://github.com/electron-userland/electron-builder) to package our apps into native format depending on OS.
 
-`npm start`
+1. `npm run insall` to install all node modules from npm and yarn
```

Comment:
typo `npm run install`

Reply:
why not include `npm install` in `npm run build:xxx`

Reply:
changed to `npm run install-dep`